src/config.jsx

```diff
@@ -2,7 +2,7 @@ import Head from 'next/head'
 import Script from 'next/script'
 
 const titleDefault = 'Kitty Inu DAO'
-const url = 'https://kittyinudao.eth.link/'
+const url = 'https://kittyinudao.eth.limo'
```

Comment:
should be `https://kitty-inu-dao.eth.limo/`

Reply:
Done.